x/ibc-hooks/client/cli/query.go

```diff
@@ -0,0 +1,56 @@
+package cli
```

Comment:
can we use osmocli for this instead?

Reply:
I started looking into that and realized I didn't know how to use osmocli 😅, so in the interest of time, I just went with what I knew